pair-finder/pair_finder/pipeline/steps/get_jobs_from_travis_api.py

```diff
@@ -72,6 +74,20 @@ def process(self, data: Any, context: dict) -> Optional[Any]:
         #   WHERE j.repo_id = "<repo_id>"
         jobs = []
         for build in build_list:
```

Comment:
Instead of line24 `self.last_date_mined = self.set_current_date()`, can we get the latest date time from the build since we are iterating the build?